eva/binder/statement_binder.py

```diff
@@ -147,6 +147,15 @@ def _bind_select_statement(self, node: SelectStatement):
             self.bind(node.union_link)
             self._binder_context = current_context
 
+        assert not (
+            self._binder_context.is_retrieve_audio()
+            and self._binder_context.is_retrieve_video()
+        ), "Cannot query over both audio and video streams"
```

Comment:
To make sure my understanding is correct, does this mean we can not query audio and video in the same query?

Reply:
Yes, we are going with this design for now. We plan to support both in the future if we find a use case.

Reply:
Got it. We can explore a JOIN-like syntax for video and audio in the future. For example,  `SELECT ... FROM VIDEO, AUDIO WHERE ...`.